blogposts/2021/release-3.25.md

```diff
@@ -27,5 +27,8 @@ changelogItems:
   - description: "When previewing a campaign spec, changesets can be filtered by current state or the action(s) to be performed."
     url: https://github.com/sourcegraph/sourcegraph/issues/16960
     category: Campaigns
+  - description: "Experimental: Sync Perforce depots directly through the Sourcegraph UI. To enable, use the feature flag `\"experimentalFeatures\": { \"perforce\": \"enabled\" }`. For more information on how to add your Perforce depots, please reference our docs (https://docs.sourcegraph.com/admin/repo/perforce)."
```

Comment:
Since the docs are what are linked on the entire row, I don't think we need to include them in the wording. What do you think?

```suggestion
  - description: "Experimental: Sync Perforce depots directly through the Sourcegraph UI. To enable, use the feature flag `\"experimentalFeatures\": { \"perforce\": \"enabled\" }`."
```